### ISSUE TITLE
Book 05: Sentence 058

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#9]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/960d63db-bddc-4c12-9868-34632a4a0cfc)

```
# sent_id = Book_05_-_The_Omens.txt:58
# text = Injc muchichiuh tochin, nacazpapatlactic, tentipontic, tentetepontic, eliztac: ioan çan cujtlapiltetepontic.
# text[norm] = Inic mochichiuh tochin, nacazpapatlactic, tentipontic, tentetepontic, eliztac: ihuan zan cuitlapiltetepontic.
# text[orig] = Injc·muchichiuh·tochin·,·nacazpapatlactic·,·ten·tipontic·,·tentete¶pontic·,·eliztac·:·ioan·çan·cujtlapiltete¶pontic·.
# text[osp] = Los conejos desta tierra, son como los de españa: aunque no tienen tan buen comer.
# references = A&D/171:3
1	Injc	inic	SCONJ	_		2	_	_	Folio=8v|Paragraph=6|Line=7|Norm=Inic|Analysed=Yes
2	muchichiuh	chichihua	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Past|VerbForm=Fin	0	root	_	Folio=8v|Paragraph=6|Line=7|Norm=mochichiuh|Analysed=Yes
3	tochin	tochin	NOUN	_	State=Abs	2	_	_	Folio=8v|Paragraph=6|Line=7|Norm=tochin|AmbigNorm=True|Override=False|Gloss=conejo|Analysed=Yes
4	,	,	PUNCT	_		5	punct	_	Folio=8v|Paragraph=6|Line=7|Norm=,|Analysed=Yes
5	nacazpapatlactic	nacazpapatlactic	ADJ	_		2	_	_	Folio=8v|Paragraph=6|Line=8|Norm=nacazpapatlactic|AltPos=VERB|Split=nacaz·papatlactic|Analysed=Yes
6	,	,	PUNCT	_		7	punct	_	Folio=8v|Paragraph=6|Line=8|Norm=,|Analysed=Yes
7	tentipontic	tentipontic	ADJ	_		5	_	_	Orig=ten·tipontic|Folio=8v,8v|Paragraph=6,6|Line=8,8|Norm=tentipontic|AltPos=VERB|Split=ten·tipontic|Analysed=Yes
8	,	,	PUNCT	_		9	punct	_	Folio=8v|Paragraph=6|Line=8|Norm=,|Analysed=Yes
9	tentetepontic	tentetepontic	ADJ	_		5	_	_	Orig=tentete¶·pontic|Folio=8v,8v|Paragraph=6,6|Line=8,9|Norm=tentetepontic|AltPos=VERB|Split=ten·tetepontic|Analysed=Yes
10	,	,	PUNCT	_		11	punct	_	Folio=8v|Paragraph=6|Line=9|Norm=,|Analysed=Yes
11	eliztac	eliztac	ADJ	_		5	_	_	Folio=8v|Paragraph=6|Line=9|Norm=eliztac|Gloss=ser.blanco|Split=el·iztac|Root=el|Analysed=Yes
12	:	:	PUNCT	_		13	punct	_	Folio=8v|Paragraph=6|Line=9|Norm=:|Analysed=Yes
13	ioan	huan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Poss	5	_	_	Folio=8v|Paragraph=6|Line=9|Norm=ihuan|Gloss=con|Analysed=Yes
14	çan	zan	ADV	_		15	_	_	Folio=8v|Paragraph=6|Line=9|Norm=zan|Analysed=Yes
15	cujtlapiltetepontic	cuitlapiltetepontic	ADJ	_		13	_	_	Orig=cujtlapiltete¶·pontic|Folio=8v,8v|Paragraph=6,6|Line=9,10|Norm=cuitlapiltetepontic|AltPos=VERB|Split=cuitla·pil·tetepontic|Analysed=Yes
16	.	.	PUNCT	_		2	punct	_	Folio=8v|Paragraph=6|Line=10|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
Probably uncontroversial:
```
nsubj(mochichiuh, tochin)
advmod(cuitlapiltetepontic, zan)
```

Questions:

López Austín has:
> Así se atavía el conejo: orejas anchas, labios retorcidos, labios hendidos, pecho blanco y el rabo hendido.

- Are the `ADJ`s clauses? e.g. *es de orejas anchas* or nominals *orejasanchas*. 
  - Are we looking at `parataxis` or `conj`? 
- Is the root *mochichiuh* or *nacazpapatlactic*?
- Is the initial *inic* a subordinator `SCONJ` or adverb? [Check Launey]

**Comment by Lguyogiro** [comment]
- I'm confused about the use of "chichihua" in this sentence. 
- Lockhart has a pretty thorough discussion of _inic_, wherein he mentions a use meaning ~ "how", like _inic mochihua molli_ "how the mole is made". This seems sort of similar to the present sentence, something like "thus how the rabbit is made/marked/etc", so I would lean towards `ADV`

**Comment by Lguyogiro** [comment]
> Is the root mochichiuh or nacazpapatlactic?
I vote for _mochichiuh_, being the first verb and, since above I argued for _inic_  not being a subordinator.

> Are the ADJs clauses? e.g. es de orejas anchas or nominals orejasanchas. 

I don't think there is much here to help us distinguish these two analyses, except for where how we choose the head of the first one. i.e. if the head of _nacazpapatlactic_ is _mochichiuh_, we should call them clauses (maybe `parataxis(mochichiuh, nacazpapatlactic)`).

**Comment by ftyers** [comment]
Sounds good to me. Final tree:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/f1105342-e75c-42f0-897b-536edf1f5979)


Fixed in 64ff6a5.
